firefox_ui_harness/runtests.py

```diff
@@ -4,18 +4,37 @@
 
 import copy
 import sys
+import tempfile
 
-from marionette import BaseMarionetteTestRunner
-from marionette.runtests import cli
+import mozinstall
 
+from marionette import (BaseMarionetteOptions,
+                        BaseMarionetteTestRunner,)
+from mozlog import structured
+
+import firefox_puppeteer
 import firefox_ui_tests
 
-from .arguments import ReleaseTestParser
 from .default_prefs import default_prefs
 from .testcase import FirefoxTestCase
 
 
-class ReleaseTestRunner(BaseMarionetteTestRunner):
+class FirefoxUIOptions(BaseMarionetteOptions):
```

Comment:
I mimic what we have in marionette base.py and what gaiatest is doing. I don't see a reason why to offload those tiny bits into various modules.